sdk/appconfiguration/Azure.Data.AppConfiguration/src/SettingSelector.cs

```diff
@@ -4,49 +4,63 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel;
-using System.Linq;
-using Azure.Core;
 
 namespace Azure.Data.AppConfiguration
 {
     /// <summary>
-    /// Set of options for selecting <see cref="ConfigurationSetting"/> from the configuration store.
+    /// <see cref="SettingSelector"/> is a set of options that allows selecting
+    /// a filtered set of <see cref="ConfigurationSetting"/>s from the
+    /// configuration store, and optionally allows indicating which fields of
+    /// each setting to retreive.
+    ///
```

Comment:
These blank lines will be canonicalized into one long paragraph. If you want to keep them, put each block in `<para></para>` tags.